paseos/actors/actor_builder.py

```diff
@@ -77,6 +76,20 @@ def set_orbit(
 
         logger.debug(f"Added orbit to actor {actor}")
 
+    def set_position(actor: BaseActor, position: list):
```

Comment:
Do we want to use the same for ground stations? If so, wouldn't it make sense to allow for 2D long/lat vectors too? Maybe the assert can be in an isinstance(SpaceActor)?

Reply:
For the ground stations I am currently thinking to have a dedicated set_position_on_Earth or such since they are not stationary. the position without orbit does not update. The ground station position will need to be updated. (Also thinking about how / whether we want ground stations on other bodies but is also stuff for #16 , would leave that out of here for now)